clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/BasicAmazonDeployHandler.groovy

```diff
@@ -455,24 +454,6 @@ class BasicAmazonDeployHandler implements DeployHandler<BasicAmazonDeployDescrip
     asgReferenceCopier.copyScheduledActionsForAsg(task, sourceAsgName, targetAsgName)
   }
 
-  @VisibleForTesting
-  @PackageScope
-  void createLifecycleHooks(Task task,
-                            RegionScopedProviderFactory.RegionScopedProvider targetRegionScopedProvider,
-                            NetflixAmazonCredentials targetCredentials,
-                            BasicAmazonDeployDescription description,
-                            String targetAsgName) {
-
-    try {
-      List<AmazonAsgLifecycleHook> lifecycleHooks = getLifecycleHooks(targetCredentials, description)
-      if (lifecycleHooks.size() > 0) {
-        targetRegionScopedProvider.asgLifecycleHookWorker.attach(task, lifecycleHooks, targetAsgName)
-      }
-    } catch (Exception e) {
-      task.updateStatus(BASE_PHASE, "Unable to attach lifecycle hooks to ASG ($targetAsgName): ${e.message}")
```

Comment:
we used to log the exception message when attaching failed but not anymore?

Reply:
Good catch, @dreynaud! Re-added the error. Please have a look and let me know.